pkg/collector/collector.go

```diff
@@ -31,7 +31,18 @@ const (
 const cancelCheckTimeout time.Duration = 500 * time.Millisecond
 
 // Collector abstract common operations about running a Check
```

Comment:
```suggestion
// Collector manages a collection of checks and provides operations over them
```

Seems like a good time to update and clarify this comment now.